src/Test/Perf/infra/fetch_build.csx

```diff
@@ -29,9 +35,16 @@ if (latestBuild == null)
 var latestBuildFolder = Path.Combine(sourceFolder, latestBuild);
 Log($"Fetching build \"{latestBuildFolder}\".");
 
-Directory.Delete(destinationFolder, recursive: true);
+if (Directory.Exists(destinationFolder))
+{
+    Directory.Delete(destinationFolder, recursive: true);
+}
 
-CopyDirectory(latestBuildFolder, destinationFolder);
+var result = ShellOut("Robocopy", $"/np /s {latestBuildFolder} {destinationFolder}", logStreamingOutput: true);
```

Comment:
Could this be ShellOutVital?

If ShellOutVital fails, it prints the command with arguments in the failure message.